test/unittests/state_transition.cpp

```diff
@@ -197,6 +197,9 @@ void state_transition::export_state_test(const TransactionReceipt& receipt, cons
         jtx["maxPriorityFeePerGas"] = hex0x(tx.max_priority_gas_price);
     }
 
+    for (size_t i = 0; i < tx.initcodes.size(); ++i)
```

Comment:
Maybe something like:
```cpp
auto&  jinitcodes = jtx["initcodes"] = json::json::array();
for (const auto& initcode : tx.initcodes)
    jinitcodes.push_back(hex0x(initcode));

Reply:
I don't like this double assignment, especially with a reference, but this is similar to other parts of this code, so I'll change it.